standalone-metastore/pom.xml

```
@@ -285,6 +285,11 @@
         <artifactId>javax.jdo</artifactId>
         <version>${datanucleus-jdo.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.springframework</groupId>
+        <artifactId>spring-context</artifactId>
+        <version>${springframework.version}</version>
+      </dependency>
```

I don't know how `spring-context` was chosen here. It is probably better to use the spring BOM to enforce ALL spring dependencies to a single version. This works just like an individual dependnecy in dependencyManagement, but does it for all the spring dependencies. Maven boms exist for netty, spring, jetty, and some other common libraries.

```suggestion
      <dependency>
        <groupId>org.springframework</groupId>
        <artifactId>spring-framework-bom</artifactId>
        <version>${springframework.version}</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
```